### DISCUSSION TITLE
Repositories

### DISCUSSION THREAD

**Discussion by Dhyros** [community/community#50280] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Some time ago I created a repository through another repository of a person who deleted it. Since then, I don't have that repository visible from the "Your repositories" section, so I can't delete it.
Does anyone know how I can delete it?

**Comment by bbkx226** [comment]
If you have forked a repository from someone else's account, and the original repository has been deleted, then your forked repository may still exist, but it may not be visible in your repository list. Here are the steps you can follow to delete the repository. First, go to the URL of the repository you want to delete. The URL should be in the format: https://github.com/ username / repo name. Click on the "Settings" tab in the repository menu, then scroll down to the bottom of the page, and click on the "Delete this repository" button. Follow the prompts to confirm that you want to delete the repository.

Then I think that's all, let me know if you've resolved the probelm.

**Comment by Anmol-Baranwal** [comment]
If you are unable to delete a repository from the "Your repositories" section on GitHub, you can try deleting it using the following steps:

1. Go to the repository's URL: If you know the URL of the repository, go to it directly.

2. Click on the "Settings" tab: This is located near the top right corner of the repository page.

3. Scroll down to the "Danger Zone" section: This section is located at the bottom of the settings page.

4. Click on the "Delete this repository" button: This will prompt you to confirm that you want to delete the repository.

5. Confirm the deletion: Type the name of the repository into the text field, then click the "I understand the consequences, delete this repository" button.

> This might resolve your issue.

**Comment by Dhyros** [comment, marked as answer]
I can't remember the name of the repository. Is there any way to find out?

**Reply by Anmol-Baranwal** [reply]
Yes, on GitHub there are several ways to find a repository even if you can't remember its name. You can try these steps:

1. Use GitHub's search feature: On the GitHub homepage, you can use the search bar to search for repositories using keywords related to the repository's name, description, or contents. You can filter the results by repository type, language, and other criteria.

2. Look through your account activity: If you have contributed to the repository or interacted with it in any way, you can check your account activity to see if you can find a reference to it. Go to your profile, click on "Activity" and then select "Repositories" from the dropdown menu to see a list of repositories you have interacted with.

3. Check your starred repositories: If you have starred the repository, you can find it by going to your profile, clicking on "Stars" and then browsing through the list of repositories you have starred.

4. Look through your browser history: If you have visited the repository on GitHub before, you can try searching your browser history to see if you can find the URL of the repository.

> I hope one of these methods will help you find the repository you're looking for.

**Reply by Anmol-Baranwal** [reply]
@Dhyros 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Dhyros ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
